apps/meteor/server/startup/migrations/v304.ts

```diff
@@ -0,0 +1,9 @@
+import { upsertPermissions } from '../../../app/authorization/server/functions/upsertPermissions';
```

Comment:
You don't need to create migrations for permissions anymore. `upsertPermissions` is called automatically when the server starts after an update.

Reply:
Done!